unit-tests/live/safety_preset/test-get-set-xu-cmd.py

```diff
@@ -0,0 +1,62 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+# test:device S585
+import pyrealsense2 as rs
+from rspy import devices, log, test, file, repo
+
+#############################################################################################
+# Tests
+#############################################################################################
+
+test.start("Init")
```

Comment:
Maybe verify safety sensor extension?

Reply:
I meant changing the test name to `verify safety sensor extension` rather than `Init`
What do you think?

Reply:
done